src/decision/subroutines/include/Subroutine.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: March 17, 2018
+ * Description:
+ */
+#ifndef DECISION_SUBROUTINE_H
+#define DECISION_SUBROUTINE_H
+
+#include <geometry_msgs/Twist.h>
+#include <ros/ros.h>
+
+#define RIGHT -1.0
+#define LEFT 1.0
+#define FORWARD 1.0
+#define BACKWARD -1.0
+#define UP 1.0
+#define DOWN -1.0
```

Comment:
Why are we defining these here? What are they used for?

Reply:
They're used by all the subroutines when creating the output twist message, such that we can read the logic more easily of what they are doing. So you get:
```z_rotation = RIGHT```
rather than
```z_rotation = -1.0```
and have to figure out yourself what direction its trying to move

Reply:
Ah, that makes sense. We should add a comment here to that effect. Also, I think we should be using `static const` instead of `#define` here. (ie. `static const double RIGHT = -1`). This has several advantages, including being more explicit about the type of `RIGHT` and allowing for proper scoping. For differences between `#define` and `static const` see https://stackoverflow.com/questions/1637332/static-const-vs-define?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
Fair enough, changed